tests/e2e/standalone/run_test.go

```diff
@@ -129,3 +133,137 @@ func TestStandaloneRun(t *testing.T) {
 		require.Contains(t, output, "The id for your application, used for service discovery", "expected usage to be printed")
 	})
 }
+
+func TestStandaloneRunNonDefaultDaprPath(t *testing.T) {
+	// these tests timeout on MacOS for some reason
+
+	t.Run("run with flag", func(t *testing.T) {
+		if runtime.GOOS == "darwin" {
```

Comment:
IMO this needs to be investigated for why it is timing out.

Reply:
@pravinpushkar agreed but unfortunately i do not have access to Apple hardware.

Reply:
@dapr/approvers-cli @dapr/maintainers-cli should check out your PR locally and investigate. For now I recommend you do not disable macOS tests.